plugins/git/git.plugin.zsh

```diff
@@ -215,3 +215,8 @@ alias glum='git pull upstream master'
 
 alias gwch='git whatchanged -p --abbrev-commit --pretty=medium'
 alias gwip='git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit -m "--wip--"'
+
+# Remove from repo all files deleted
+alias gitstrm="git st | egrep 'deleted' | sed -e 's/#//' -e 's/deleted:/git rm/' | sh"
+
+alias gitgcprune="git remote prune origin ; git gc --prune --aggressive"
```

Comment:
This shouldn't be an alias, it should be as hard as possible to type the command because it's a destructive command.